src/main/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre
+FROM openjdk:11-jre
```

Comment:
add flags for ignoring groovy warnings:  WARNING: An illegal reflective access operation has occurred

Reply:
done